api/validation/MetadataSchema.py

```diff
@@ -150,6 +181,7 @@ class Config:
     geo: Optional[List[GeoAnnotation]]
     date: Optional[List[DateAnnotation]]
     feature: Optional[List[FeatureAnnotation]]
+    transformations: Optional[Dict[str, Transformation]]
```

Comment:
Something in this is causing the transformation annotations to not store anymore. I am taking a look, but I think the format we are feeding the annotations in do not match this validator format.

Reply:
This all works locally for me. I'm going to mark this as resolved for now but happy to revisit if it continues to be an issue.